src/main/scala/tscfg/generators/tsConfigUtil.scala

```diff
@@ -74,6 +80,19 @@ object tsConfigUtil {
       case `day`    =>  "DAYS"
     })
 
+  def temporalUnitString(q: DurationQualification): String = {
```

Comment:
Seems like this method is not called; I don't see it in any other parts of the change set, which seems confirmed by coverage report https://coveralls.io/builds/21006451/source?filename=src%2Fmain%2Fscala%2Ftscfg%2Fgenerators%2FtsConfigUtil.scala#L84 (which decreased). If really unneeded, please remove it.